src/main/java/com/medallia/word2vec/Word2VecModel.java

```diff
@@ -214,6 +216,30 @@ public static Word2VecModel fromBinFile(File file, ByteOrder byteOrder, Profilin
 		}
 	}
 
+	public void toBinFile(final OutputStream out) throws IOException {
```

Comment:
Great! Want to add a bit of doc on this indicating that the output is in little endian format for consumption by the C impl?

Also, should the ByteOrder be a parameter?

Can we add a unit test for round trip storing and loading?

Lastly, perhaps add an example in the Word2VecExamples?